init_metadata.jl

```diff
@@ -0,0 +1,31 @@
+# Create the initial contents of `PackageDistro`.
+using SHA
+using JSON
+using DataStructures
+using HTTP
+
+info = read(expanduser("~/package-infos.json"), String)
```

Comment:
I would simply pass the filename(s) as argument to this script, via `sys.argv` (see the scripts in the GAP repo for examples).

That said, this is mostly a one-off script, isn't it? To be used possibly multiple times this week, to get initial data in?